redash/settings/__init__.py

```diff
@@ -226,6 +226,7 @@ def all_settings():
 QUERY_REFRESH_INTERVALS = map(int, array_from_string(os.environ.get("REDASH_QUERY_REFRESH_INTERVALS", "60, 300, 600, 900, 1800, 3600, 7200, 10800, 14400, 18000, 21600, 25200, 28800, 32400, 36000, 39600, 43200, 86400, 604800, 1209600, 2592000")))
 PAGE_SIZE = int(os.environ.get('REDASH_PAGE_SIZE', 20))
 PAGE_SIZE_OPTIONS = map(int, array_from_string(os.environ.get("REDASH_PAGE_SIZE_OPTIONS", "5,10,20,50,100")))
+TABLE_CELL_MAX_JSON_SIZE = 200000
```

Comment:
Let's keep previous value `50000` as default (you can override it in your setup)

Reply:
Does it make sense to get it as an environment var? :thinking: 
WDYT @kravets-levko, @arikfr ?

Reply:
@gabrieldutra Good point! Yes, let's take it from env with default `50000`